integration/docker/cpu_test.go

```diff
@@ -152,7 +152,12 @@ var _ = Describe("CPU constraints", func() {
 		quota      int = 20000
 		period     int = 15000
 		cpusetCpus int = 0
-		cpusetMems int = 0
+		// We are now not constraining the guest cgroup/cpuset
+		// values at all, e.g. we are allowing all onlined vCPUs
+		// to be used by the container. In this test, we use the
+		// hard-coded value below assuming the default vCPU is '1'
+		cpusetCpusExpected string = "0-1"
```

Comment:
if we aren't ever setting, would it be easier to just not check the cpuset cpus, and not need expected values?